src/jenkins_job_insight/main.py

```
@@ -3566,6 +3566,18 @@ async def delete_job_endpoint(
     return {"status": "deleted", "job_id": job_id}
 
 
+@app.get("/api/dashboard/active-count")
+async def get_active_analysis_count() -> dict:
+    """Get count of currently active analyses (running/pending/waiting)."""
+    logger.debug("GET /api/dashboard/active-count")
+    try:
+        count = await storage.count_active_analyses()
+        return {"count": count}
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)